lms/templates/verify_student/face_photo_step.underscore

```diff
@@ -10,18 +24,16 @@
 
```

Comment:
another reason why we should make sure this works on Mobile so we can offer at least one means of completing this process on an Apple OS.